es/utils/amount-formatter.js

```diff
@@ -1,5 +1,75 @@
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+
+/**
+ * Amount Formatter
+ * @module @aeternity/aepp-sdk/es/utils/amount-formatter
+ * @example import { format, toAettos, AE_AMOUNT_FORMATS } from '@aeternity/aepp-sdk/es/utils/amount-formatter'
+ */
+import { asBigNumber, isBigNumber } from './bignumber'
 import { BigNumber } from 'bignumber.js'
 
+const AE_MULT = 1e18
+
+/**
+ * AE amount formats
+ * @type {{AE: string, AETTOS: string}}
+ */
+export const AE_AMOUNT_FORMATS = {
+  AE: 'ae',
+  AETTOS: 'aettos'
+}
```

Comment:
Should also add the rest of the denominations. 

E.g. `picoAE`, `miliAE`, `microAE` etc.